cg/cli/workflow/mutant/base.py

```diff
@@ -100,3 +98,38 @@ def start_available(context: click.Context, dry_run: bool = False):
             exit_code = EXIT_FAIL
     if exit_code:
         raise click.Abort
+
+
+@mutant.command("store-available")
+@DRY_RUN
+@click.pass_context
+def store_available(context: click.Context, dry_run: bool) -> None:
```

Comment:
As we discussed, this implementation of a `store` command that encompasses both qc+storing for a case is not possible because of the limitation of this scenario:

- User wants to manually run QC on a case;
- Case fails qc, gets TB status `Fail` but needs to be delivered anyway;
- User then runs `store` for that case and:
	- QC is run again unnecessarily;
	- Case is set to fail;
	- Analysis is not stored.

As such, it is best to separate qc from storing as much as possible.
Ideally we would change how the pipeline is run in the future, in order for it to finish with a `QC` status instead of completed!